CONTRIBUTING.md

```diff
@@ -836,7 +835,7 @@ requirements are:
 - One [documentation team](https://github.com/orgs/kata-containers/teams/documentation/members)
   approval if the PR modifies documentation.
 
-See the `.pullapprove.yml` configuration files for full details.
+See the `.PullApprove.yml` configuration files for full details.
```

Comment:
similarly here - the files *were* lower case I think - having said that, we have nuked and dropped all remnants of `pullapprove` I think - would you rather just nuke these sections?

Reply:
I'm guessing you are reviewing "a commit at a time"? :)

Reply:
Yep, I tend to when there is >1 commit - most times I find it easier to see what has changed, but sometimes you then run into odd-ordering things :-)